onadata/apps/api/tests/viewsets/test_xform_viewset.py

```diff
@@ -4351,6 +4352,42 @@ def test_delete_xform_also_deletes_linked_dataviews(self):
         self.assertIsNotNone(self.data_view.deleted_at)
         self.assertIn("-deleted-at-", self.data_view.name)
 
+    def test_delete_xform_endpoint(self):
+        """
+        Tests that the delete_xform view soft deletes xforms
+        """
+        # publish form and make submissions
+        xlsform_path = os.path.join(
+            settings.PROJECT_ROOT, "libs", "tests", "utils", "fixtures", "tutorial.xlsx"
+        )
+        self._publish_xls_form_to_project(xlsform_path=xlsform_path)
+        for x in range(1, 9):
+            path = os.path.join(
+                settings.PROJECT_ROOT,
+                "libs",
+                "tests",
+                "utils",
+                "fixtures",
+                "tutorial",
+                "instances",
+                "uuid{}".format(x),
+                "submission.xml",
+            )
+            self._make_submission(path)
+            x += 1
```

Comment:
Why do we increment?

Reply:
There's no need for, this, I've removed this line